pkg/kritis/review/review_test.go

```diff
@@ -649,19 +649,23 @@ func TestGetAttestationAuthoritiesForISP(t *testing.T) {
 	}
 }
 
-func makeRawAttestationPgp(signature, id, occId string) metadata.RawAttestation {
+func makeRawAttestationPgp(signature, id string) metadata.RawAttestation {
```

Comment:
Is this the same as in metadata_test?  If so, then please leave a TODO to move it to a shared location.  (No need to block this PR though.)

Reply:
Qifan and I agreed to move these functions into metadata.go